Plugins/Gradle/build.gradle

```diff
@@ -188,4 +120,59 @@ task assembleLocal(description: 'Assembles the outputs of this project for use l
     copyLocal.mustRunAfter build
 }
 
-apply from: '../../ktlint.gradle'
\ No newline at end of file
+signing {
```

Comment:
Can we re-use the `publish.build.gradle` from Library? Maybe move that up a level and share it?